3 - PB/Documentazione interna/Verbali/24-03-22/24-03-22.typ

```diff
@@ -0,0 +1,34 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "22/03/24",
+  subTitle: "Meeting post colloquio con i Proponenti",
+  docType: "verbale",
+  authors: (
+    "Banzato Alessio",
+  ),
+  reviewers: (
+    "Gardin Giovanni",
+  ),
+  missingMembers: (
+    "Todesco Mattia",
+  ),
+  timeStart: "16:30",
+  timeEnd: "17:00",
+);
+
+= Ordine del giorno
+A seguito dell'incontro con i Proponenti, il gruppo ha svolto un meeting interno riguardante:
```

Comment:
Se non erro, il proponente è sempre singolare perchè individuabile come "l'Azienda". In questo caso potresti o mettere al singolare, o parlare di "referenti", just my opinion